lib/GLPI/Agent/Task/Inventory/Generic/Remote_Mgmt/RMS.pm

```diff
@@ -0,0 +1,73 @@
+package GLPI::Agent::Task::Inventory::Generic::Remote_Mgmt::RMS;
+
+use strict;
+use warnings;
+
+use parent 'GLPI::Agent::Task::Inventory::Module';
+
+use English qw(-no_match_vars);
```

Comment:
A dependency is missing here.
```suggestion
use English qw(-no_match_vars);
use UNIVERSAL::require;
```
Unless you prefer include the dependencies from l.57 & l.58 in the case you only want to support `MSWin32` (see also my comment below).